src/functions/avl-subscriber/index.ts

```diff
@@ -73,7 +73,7 @@ export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayPr
             shortDescription: avlSubscribeMessage.shortDescription,
             requestorRef: avlSubscribeMessage.requestorRef,
             publisherId: avlSubscribeMessage.publisherId,
-            apiKey: randomUUID(),
+            apiKey: randomUUID().replaceAll("-", ""),
```

Comment:
the avl-update-endpoint has api key generation as well - at which point it's worth refactoring the api key gen into a util func and having both lambdas use it

Reply:
good shout :)